tests/search/nearest_neighbor/nearest_neighbor.rb

```diff
@@ -39,9 +39,30 @@ def run_test_nearest_neighbor_operator(mixed)
   end
 
   def test_nearest_neighbor_operator_mixed_multipoint
+    sd_file = selfdir + "mixed/test.sd"
+    run_test_nearest_neighbor_operator_mixed_multipoint(sd_file)
+  end
+
+  def test_nearest_neighbor_operator_mixed_fast_rank_multipoint
+    sd_dir = selfdir + "mixed_fast_rank"
+    sd_file = sd_dir + "/test.sd"
+    FileUtils.rm_rf(sd_dir)
+    FileUtils.mkdir_p(sd_dir)
+    origfile = File.open(selfdir + "mixed/test.sd" , "r")
+    lines = origfile.readlines
+    origfile.close
+    lines.each do |line|
+      line.sub!("# attribute:", "attribute:")
+    end
+    modfile = File.open(sd_file, "w")
+    modfile.write(lines.join)
+    modfile.close
```

Comment:
Consider moving the code that prepares the sd_file to a separate function.

Reply:
Done.